doc/source/api.rst

```diff
@@ -278,9 +278,25 @@ Attributes
    Series.size
    Series.strides
    Series.itemsize
+   Series.axes
    Series.base
    Series.T
    Series.memory_usage
+   Series.hasnans
+   Series.flags
+   Series.empty
+   Series.dtypes
+   Series.ftypes
+   Series.as_matrix
+   Series.data
+   Series.get_dtype_counts
+   Series.get_ftype_counts
+   Series.get_values
+   Series.is_copy
+   Series.imag
+   Series.real
+   Series.name
+   Series.put
```

Comment:
I have the feeling that we don't want to include all of those here (eg imag, real). If that is needed for getting rid of warnings, we can put them in comment?)

Reply:
I didn't look closely in this last round. Happy to move them to the bottom hidden section.

Reply:
And yes, it's for warnings. Every method of a class that uses the default class template (with autosummary) will need to be listed in a toctree. That's why I've added the `:hidden:` toctree at the bottom. I'll also update contributing.rst with some details.